pkg/databind/pkg/databind/cache.go

```diff
@@ -78,6 +81,24 @@ func (d *gatherer) do(now time.Time) (interface{}, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	if dataWithTTL, ok := vals.(ValuesWithTTL); ok {
+		ttl, err := dataWithTTL.TTL()
+		if err != nil && !errors.Is(err, ErrTTLNotFound) {
+			return nil, multierr.Append(ErrTTLInvalid, err)
```

Comment:
couldn't we just use fmt with %w for returning the error?

Reply:
@cristianciutea changed in 31474846517c36f31c302463e186c1233ef2f40e